langchain/prompts/prompt.py

```diff
@@ -16,6 +15,13 @@
 
 
 def _get_jinja2_variables_from_template(template: str) -> Set[str]:
+    try:
+        from jinja2 import Environment, meta
+    except ImportError:
+        raise ValueError(
```

Comment:
Nit: This should be raised as an `ImportError`, not a `ValueError`.

Reply:
I was in a dilemma of whether to use `ImportError` or `ValueError`. I used `ValueError` because this error will only raise when user specify `jinja2` as the value for `template_format` argument.

Reply:
Nvm I think `ImportError` is more appropriate, since technically it is a valid value, just package not installed yet.

Reply:
Yup, exactly, thanks for updating :).